pkgs/development/compilers/sagittarius-scheme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, libffi
+, boehmgc
+, openssl
+, zlib
+, odbc ? true
+, libiodbc
+}:
+
+let version = "0.9.6";
+    odbcDeps = if odbc then [ libiodbc ] else [];
+in stdenv.mkDerivation {
+  name = "sagittarius-scheme-${version}";
+  src = fetchurl {
```

Comment:
`fetchurl` -> `fetchFromBitbucket`. See https://nixos.org/nixpkgs/manual/ for details.

Reply:
The distribution tarball available includes the auto-generated .c files which are not included in the repository. Building directly from repository require a Sagittarius bootstrap which is not needed if you build from a distribution tarball.

Let me know if this is a good enough excuse to keep it this way.

Reply:
This is an entirely valid reason. Thanks for clarifying.